### ISSUE TITLE
Additional accessibility information missing IDs in guidelines

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#493]
The Additional accessibility information section  has six examples as descriptive and compact. There are no IDs, and no array structure .

It seems that in the techniques we should remove the arrays and list the compact strings. We can add the ID information to the guidelines for the six items. The descriptive strings would be in the JSON file only.

We should review the six items in the examples to reaffirm that those are the ones we want.

Is this the direction we want to go?

**Comment by gregoriopellegrino** [comment]
The _Additional accessibilities information_ are not uniform between ONIX techniques and EPUB Accessibility techniques... In my opinion better to leave as it is in the Principles and fix only the techniques
